templates/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
Nice job designing a page displaying the table of availability, appreciate the ownership here!